test/src/test/StringSearch.java

```diff
@@ -0,0 +1,32 @@
+package test;
+
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+
+public class StringSearch {
+
+	public static void main(String[] args) {
+		try {
+			//Hej
```

Comment:
maybe look at error checking for the args passed (e.g. incorrect number)

Reply:
Thanks. I have adjusted it now